src/test/java/com/acrolinx/sidebar/swing/AcrolinxSidebarSwingTest.java

```diff
@@ -0,0 +1,58 @@
+package com.acrolinx.sidebar.swing;
+
+import com.acrolinx.sidebar.AcrolinxIntegration;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.awt.event.KeyEvent;
+
+class AcrolinxSidebarSwingTest {
+
+    @Test
+    void processKeyEvent_PasteOnMacIsConsumed() {
+        // Given
+        KeyEvent keyEvent = Mockito.mock(KeyEvent.class);
+        AcrolinxIntegration acrolinxIntegration = Mockito.mock(AcrolinxIntegration.class);
+        AcrolinxSidebarSwing acrolinxSidebarSwing = new AcrolinxSidebarSwing(acrolinxIntegration);
+
+        // When
+        Mockito.when(keyEvent.getKeyCode()).thenReturn(KeyEvent.VK_V);
+        Mockito.when(keyEvent.getModifiers()).thenReturn(KeyEvent.META_MASK);
+
+        // Then
+        acrolinxSidebarSwing.processKeyEvent(keyEvent);
+        Mockito.verify(keyEvent, Mockito.atLeast(1)).consume();
+    }
+
+    @Test
+    void processKeyEvent_PasteOnWindowsIsConsumed() {
+        // Given
+        KeyEvent keyEvent = Mockito.mock(KeyEvent.class);
```

Comment:
all three test methods look fairly similar, could we extract some private method to reduce code redundancy?

Reply:
Yes I can do so. The part where we were to make tests as independent as possible is what I was following.

Reply:
Having a private static helper method still means we _could_ run the three tests in parallel, if we want to. The main principle is to avoid any mutable static fields and any non-static fields (if possible), which your new test class already does, so that is all good.